src/content/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator.mdx

```diff
@@ -72,199 +85,263 @@ helm upgrade --install k8s-agents-operator k8s-agents-operator/k8s-agents-operat
 
 For a complete list of configuration options, see the [README](https://github.com/newrelic/k8s-agents-operator/tree/main/charts/k8s-agents-operator#values) chart.
 
-## Monitored namespaces [#monitored-namespaces]
+## Configure auto-instrumentation
+
+After APM Auto-attach is all set up in your cluster, the next step is just to roll out the configs required to get it operational. 
+That involves having at least one instrumentation Custom Resource (CR) active in the cluster.
+
```

Comment:
This is confusing me a lot. This is an "operator" in my mind and when we refer this as "APM Auto-attach" it is confusing. We should change this to "After the operator is all set up...".